### DISCUSSION TITLE
When I search app, show me warn

### DISCUSSION THREAD

**Discussion by LWF666** [ScoopInstaller/Scoop#5745] [Help and Questions] [answered]
like this:
![image](https://github.com/ScoopInstaller/Scoop/assets/44828065/85f3218c-65bf-428a-9c8d-03758979c05a)

**Comment by HUMORCE** [comment, marked as answer]
The output clearly points out the problem: Bucket .SM contains many manifests that cannot be parsed correctly

If you feel overwhelmed, move the `.SM` directory to the desktop then try again.

**Reply by LWF666** [reply]
Thanks, bro!
